### DISCUSSION TITLE
Hide list form on startup and add edit button next to new button to display list form.

### DISCUSSION THREAD

**Discussion by shortsam1** [serenity-is/Serenity#5631] [Help] [answered]
I am new to serenity and .net core. I am using the latest .net 5.0 and the latest version of serenity.

Here is a copy of the form I am in the process of writing.
![image](https://user-images.githubusercontent.com/47765946/114630108-2ec12180-9cfd-11eb-9635-7ef7e174785f.png)

I have been asked to hide the list form (purple box) on startup of our system and only show the heading, search box and 3/4 buttons at the top (red box). As I am new to this, I have not been able to work this out and I am on a tight deadline.

They also would like to have a "Edit Report" button (pink text) at the top next to the "New Report" button. On click of this button they would like to display the list form (purple box) that is currently displayed in the pic above

All help is greatly appreciated as I am on a really tight deadline.

**Comment by minhhungit** [comment, marked as answer]
@shortsam1 if you are on deadline you can hire me 😁 , hehe, just kidding

To remove button you can add code in grid.ts file, check sample code [here](https://github.com/serenity-is/Serene/blob/master/Serene/Serene.Core/Modules/BasicSamples/Grids/RemovingAddButton/RemovingAddButton.ts#L15)
That code shows how to remove button, you can add new button like this:

```javascript
buttons.push({
      title: 'YOUR CUSTOM BUTTON',
      cssClass: 'your-custom-button',
      icon: "fa-home",
      onClick: () => {
         // do something here
      }
  });
```
To hide grid (purple box) you can use this code:

```typescript
constructor(container: JQuery) {
    super(container);

    this.element.find(".grid-container").hide(); // add this line
}
```

good luck
